src/current/v24.2/stream-a-changefeed-to-amazon-msk-serverless.md

```diff
@@ -0,0 +1,296 @@
+    ~~~
+
+    If you need further detail on setting up the Kafka client, refer to the [AWS setup guide](https://docs.aws.amazon.com/msk/latest/developerguide/create-serverless-cluster-client.html).
+1. Move to the directory of your Kafka installation:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd kafka_2.12-2.8.1/bin
+    ~~~
+
+1. It is necessary to create topics manually for MSK Serverless clusters. To create a topic, run the following:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ~/kafka_2.12-2.8.1/bin/kafka-topics.sh --bootstrap-server {msk serverless endpoint} --command-config client.properties --create --topic {users} --partitions {1}
+    ~~~
+
+    Replace:
+    - `{msk serverless endpoint}` with your endpoint copied in [Step 1.4](#step-1-create-an-msk-serverless-cluster).
+    - `{users}` with your topic name. This tutorial will use the CockroachDB `movr` workload and will run a changefeed on the `movr.users` table.
+    - `{1}` with the number of partitions you require.
+
+    You will receive confirmation output:
+
+    ~~~
+    Created topic users.
+    ~~~
+
+## Step 5. Start the changefeed
+
+In this step, you'll prepare your CockroachDB cluster to start the changefeed.
+
+1. (Optional) On the EC2 instance running CockroachDB, run the [Movr]({% link {{ page.version.version }}/movr.md %}) application workload to set up some data for your changefeed.
+
+    Create the schema for the workload:
+
+     {% include_cached copy-clipboard.html %}
+     ~~~shell
+     cockroach workload init movr
+     ~~~
+
+     Then run the workload:
+
+     {% include_cached copy-clipboard.html %}
+     ~~~shell
+     cockroach workload run movr --duration=1m
+     ~~~
+
+1. Start a SQL session. For details on the available flags, refer to the [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}) page.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --insecure
+    ~~~
+
+1. Set your organization name and [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) key:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING cluster.organization = '<organization name>';
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING enterprise.license = '<secret>';
+    ~~~
+
+1. Enable the `kv.rangefeed.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING kv.rangefeed.enabled = true;
+    ~~~
+
+1. Create an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) for the MSK Serverless URI. External connections define a name for an external connection while passing the provider URI and query parameters:
```

Comment:
consider adding a note that you can also just specify the sink uri in the feed directly, without having to go through an external connection.